core/model/legacyEvents.go

```diff
@@ -95,36 +95,28 @@ type WebToolsEvent struct {
 
 // LegacyEvent wrapper
 type LegacyEvent struct {
-	AllDay                bool            `json:"allDay" bson:"allDay"`
```

Comment:
Why have you modified this structure and remove fields. This is used by all kinds of sources - webtools etc. The feedback for removing was targeting only the TPS API params.